backend/compact-connect/lambdas/provider-data-v1/tests/function/test_data_model/test_provider_transformations.py

```diff
@@ -111,19 +124,32 @@ def test_transformations(self):
             expected_license = json.load(f)
         with open('../common-python/tests/resources/dynamo/privilege.json') as f:
             expected_privilege = json.load(f)
+        with open('../common-python/tests/resources/dynamo/military-affiliation.json') as f:
+            expected_military_affiliation = json.load(f)
+            # in this case, the status will be initializing, since it is not set to active until
+            # the military affiliation document is uploaded
+            expected_military_affiliation['status'] = 'initializing'
 
         # Force the provider id to match
```

Comment:
Is this comment accurate?

Reply:
yes, because the test only calls 'create_military_affiliation' and not the s3 event trigger, the military affiliation record will be in an 'initializing' state.

Reply:
ah sorry, I was looking at the lines about this, no that comment looks outdated, I'll remove it.